### PR TITLE
allow shop-review-interfaces block on store.order-placed

### PR DESCRIPTION
**Pull request by mvandeusen**
allow new block on store.order-placed

#### What is the purpose of this pull request?

adding new allowed block to store interfaces

#### What problem is this solving?

This block allows the block defined in the shop-review-interfaces app to add components to the order-placed page
How should this be manually tested?

https://marc--sandboxusdev.myvtex.com/checkout/orderPlaced/?og=933751548681 will display pop-up shopper approved rating survey on order-placed.

####Screenshots or example usage
image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store",
-  "version": "2.22.4",
+  "version": "2.21.4",
```
Comment (brunojdo) [inline manifest.json]: I think you need to rebase this branch

Comment (mvandeusen) [reply manifest.json]: I think I've successfully rebased this branch now, I notice the file conflict warning in the pull request has gone away. This is my first time using github so please let me know if I've done something wrong or if you have any more suggestions for things I need to do to my branch.

Comment (klzns) [reply manifest.json]: Sorry for the delay, but I'm afraid you need to rebase again.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,8 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [2.23.0] - 2019-05-22
+## [2.23.1] - 2019-05-23
```
Comment (klzns) [inline CHANGELOG.md]: You don't need to add this

File: manifest.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store",
-  "version": "2.23.0",
+  "version": "2.23.1",
```
Comment (klzns) [inline manifest.json]: Do not update the version

File: manifest.json [hunk 4]
```diff
@@ -36,7 +36,9 @@
     "vtex.order-placed": "1.x",
     "vtex.pixel-manager": "0.x",
     "vtex.rich-text": "0.x",
-    "vtex.flex-layout": "0.x"
+    "vtex.flex-layout": "0.x",
+    "vtex.sandbox": "0.x"
```
Comment (klzns) [inline manifest.json]: Missing `,`

File: react/.eslintrc [hunk 5]
```diff
@@ -1,11 +1,8 @@
 {
-  "extends": "vtex-react",
+  "extends": "eslint-config-vtex",
```
Comment (klzns) [inline react/.eslintrc]: Keep how it was

File: react/yarn-error.log [hunk 6]
```diff
@@ -0,0 +1,5758 @@
+Arguments:
```
Comment (klzns) [inline react/yarn-error.log]: Remove this file

File: store/interfaces.json [hunk 7]
```diff
@@ -76,6 +77,7 @@
   },
   "store.orderplaced": {
     "required": ["order-placed"],
+    "allowed":["shop-review-form"],
```
Comment (klzns) [inline store/interfaces.json]: nitpick
```suggestion
    "allowed": ["shop-review-form"],
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -7,8 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### Added
+- Add support for `sandbox`.
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- Support for `sandbox`.
```

File: react/ProfileChallenge.js [hunk 9]
```diff
@@ -19,52 +19,49 @@ function useSafeState(initialState) {
...
+        } else {
+          return Promise.reject()
+        }
+      })
+      .catch(() => {
+        setLoading(false)
+        setLoggedIn(false)
+        redirectToLogin()
+      })
+  }, [page, redirectToLogin, setLoading, setLoggedIn])
```
Comment (jgfidelis) [inline react/ProfileChallenge.js]: no need to put setLoading and setLoggedIn here I believe

**Pull request merged**